.pipelines/mac-ios-spm-release-validation-pipeline.yml

```diff
@@ -14,10 +14,40 @@
     - template: templates/use-xcode-version.yml
       parameters:
         xcodeVersion: ${{ variables.xcodeVersion }}
+    
+    # Note: Currently it requires a dev version extensions c pod for testing
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'specific'
+        project: 'Lotus'
+        definition: 1206          #'definitionid' is obtained from `System.DefinitionId` of extensions CI: extensions.ios_packaging
+        buildVersionToDownload: 'specific'
+        branchName: 'yguo/change-header-path'   # TODO: update branch name here
+        pipelineID: 331190
+        targetPath: '$(Build.ArtifactStagingDirectory)'
 
     - script: |
         set -e -x
-        xcodebuild test -scheme onnxruntime -destination 'platform=iOS Simulator,name=iPhone 14'
+        ls
+      workingDirectory: '$(Build.ArtifactStagingDirectory)/ios_packaging_artifacts'
+      displayName: "List staged artifacts for ORT Ext C Pod"
+
+    # Note: Running xcodebuild test on `onnxruntime-Package` scheme will perform swift tests for both OnnxRuntimeBindings
+    # and OnnxRuntimeExtensions targets.
+    - script: |
+        set -e -x
+
+        cd "$(Build.ArtifactStagingDirectory)/ios_packaging_artifacts"
+        POD_ARCHIVE=$(find . -name "pod-archive-onnxruntime-extensions-c-*.zip")
+
+        shasum -a 256 "$(Build.ArtifactStagingDirectory)/ios_packaging_artifacts/${POD_ARCHIVE}"
```

Comment:
Is there a reason to do this in the CI for validation released packages?

Reply:
in the release CI testing, the extensions c pod archive used is still a dev version for now.

but yeah maybe it's not necessary to print the checksum here.